Creating SQLite database and creating endpoint to display all users.

juliaalencarb opened the pull request:
### Features
---

Creating 'users' table using SQLite with one mock user and integrating with our app. Also creating /login/users endpoint to get all users in json format.

sord-dev commented:
Lets go DB 😎

Could we move the db.sqlite file into the /config folder? 

I see md is what we're using for the hashing? did you look into the security of the package as i thought bcrypt was the standard idrk tho c: 

alsoalso last thing what I talked about with body parser too ^^ I'm gonna add express.json() to the index file as it's less filesize, faster and built into express

juliaalencarb commented:
> Lets go DB 😎
> 
> Could we move the db.sqlite file into the /config folder?
> 
> I see md is what we're using for the hashing? did you look into the security of the package as i thought bcrypt was the standard idrk tho c:
> 
> alsoalso last thing what I talked about with body parser too ^^ I'm gonna add express.json() to the index file as it's less filesize, faster and built into express

Heya! I moved db.sqlite to /config folder, so our code is more organised :) and also removed body parser, as previously discussed. 
About the md5 module, I have it just for now as it's the one I'm used to use. But I plan to change it to bcrypt as soon as I have the post endpoint working!
